tests/integration/commands/test_creation.py

```diff
@@ -107,3 +107,17 @@ def test_topic_creation_with_template_works(
     assert config_from_template.num_partitions == num_partitions
     for config_key, value in config.items():
         assert config_from_template.config[config_key] == value
+
+
+def test_consumer_group_creation(
```

Comment:
How about another test that checks the `--partition` and `--offset` parameters?

Thinking about it... maybe it would be good to be able to specify more than one topic partition and offset. How about a multi argument that takes strings in the form:  `<topic name>([<partition>])(=<offset>)` where partition defaults to all and offset defaults to 0? E.g.:
`"mytopic"` -> set offset 0 for all partitions of topic `mytopic`
`"myotpic=10"` -> set offset 10 for all partitions of topic `mytopic`
`"mytopic[0]=10"` -> set offset 10 for partition 0 of topic `mytopic`